person.rb

```diff
@@ -18,4 +21,8 @@ def can_use_services?
   def of_age?
     age >= 18
   end
+
+  def correct_name
+    @name
+  end
```

Comment:
- [ ] **[OPTIONAL]** This should not be a private method. I had to make it public before I could test your app.

  ![image](https://github.com/Ayokunnumi1/school-library/assets/36542986/336bfb2f-2695-40ce-b4ff-bf3e6277d062)

Reply:
Thanks for your corrections, I have resolved them